Dshop/conftest.py

```diff
@@ -1,15 +1,46 @@
+import datetime
 import pytest
 from dj_shop_cart.cart import get_cart_class
 from django.urls import reverse
 from pytest import fixture
 from apps.products_catalogue.models import Category, Product
-from apps.users.models import CustomUser
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
 
 
-User = CustomUser
+User = get_user_model()
 Cart = get_cart_class()
 
 
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+@pytest.fixture
+def api_client_authed():
```

Comment:
Could you please change to full name? api_client_authenticated? It makes it more readable. 
Also, the previous fixture, authenticated_api_client - why it is not used anymore? Seemed ok.

Reply:
authenticated_api_client caused integrity error - unique constraint violated in this place.
With my fixture test passed.

Also my fixture is simpler - no playing with Token

Reply:
Also authenticated_api_client (old) is not located in a good place. It's hard to share it across different apps

Reply:
Regarding fixtures --> there is little bit of mess here.

There are knop-k's fixtures:
create_active_product
create_inactive_product

mkrzminski's:
tv_product

my:
inactive_product


Also there are:
api_client (bare) in Dshop/conftest.py
api_client (authed)Dshop/apps/products_catalogue/tests/conftest.py

yours
products in Dshop/apps/products_catalogue/tests/conftest.py

mine:
ten_tv_products in Dshop/conftest.py

Reply:
We need ALWAYS to check existing fixtures before creating new ones

Reply:
ok, so what is your final response? I see a lot of text, but I don't see anything actionable.

Reply:
My reponse is:
I'll change name.
I'd like to remove old authenticated_api_client fixture

I'm asking what to do about other duplicated fixtures (probably in new PR)

Reply:
Remove not needed code, new pr is ok.